drivers/st25r3911b/Kconfig

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menuconfig ST25R3911B_LIB
+	bool "NFC ST25R3911B library"
+	help
+	  Enable the NFC ST25R3911B library
+
+if ST25R3911B_LIB
+
+config ST25R3911B_SPI_FREQ
+	int "ST25R3911B SPI frequency"
+	default 4000000
+	help
+	  ST25R3911B SPI frequency
+
+config ST25R3911B_IRQ_PIN
+	int "ST25R3911B IRQ pin"
+	default 3
+	help
+	  Configure an IRQ pin for ST25R3911B
+
+config ST25R3911B_SS_PIN
```

Comment:
It would be better (and actually easier) to use the `cs-gpios` property of the SPI node in DeviceTree for this.

Reply:
Ok. Changed